peekingduck/nodes/model/efficientdet.py

```diff
@@ -59,6 +60,10 @@ class Node(AbstractNode):
         weights_parent_dir (:obj:`Optional[str]`): **default = null**. |br|
             Change the parent directory where weights will be stored by
             replacing ``null`` with an absolute path to the desired directory.
+        use_jit (:obj:`bool`): **default = True**. |br|
```

Comment:
This should be `default = False`.
If there is a way to auto-gen the default values based on the model's `.yml` configuration, then it will avoid discrepancies whenever the `.yml` contents are updated.

Reply:
Changed